Monika After Story/game/updates.rpy

```diff
@@ -84,9 +81,10 @@ init python:
         if topicID in Event.INIT_LOCKDB:
             Event.INIT_LOCKDB.pop(topicID)
 
-
     def mas_transferTopic(old_topicID, new_topicID, per_eventDB):
-        """DEPREACTED
+        """
+        DEPREACTED
```

Comment:
move this back up because the documentation parser will mark deprecations based on the first line.